### PR TITLE
bump wipac-rest-tools

### PR DESCRIPTION
**Pull request by dsschult**

### CODE REVIEW

File: lta/__init__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 # is zero for an official release, positive for a development branch,
 # or negative for a release candidate or beta (after the base version
 # number has been incremented)
-__version__ = "0.42.0"
+__version__ = "0.41.33"
```
Comment (blinkdog) [inline lta/__init__.py:12]: Waaaa? :question:

**Review by blinkdog — CHANGES_REQUESTED** [review]
This is a little strange...

File: lta/__init__.py (hunk 1 above)
Comment (dsschult) [reply lta/__init__.py:12]: So the semantic release tool really didn't like that the master branch code version was ahead of the github tag number.  They're supposed to be equal I guess?

Comment (blinkdog) [reply lta/__init__.py:12]: Okay. I guess it doesn't matter so much, it just seemed weird for the version to decrease.

Want to bring my change from #279 in to this PR, and we can close mine and merge this one?

Comment (blinkdog) [reply lta/__init__.py:12]: :facepalm: I missed it the first time.

**Review by blinkdog — APPROVED** [review]
Let's go! :smile:

**Pull request merged**